tiledb/sm/c_api/tiledb.h

```diff
@@ -5287,6 +5287,29 @@ TILEDB_EXPORT int32_t tiledb_vfs_copy_file(
     const char* old_uri,
     const char* new_uri);
 
+/**
+ * Copies a directory. If the destination directory exists, it will be
+ * overwritten.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * tiledb_vfs_copy_dir(
+ * ctx, vfs, "hdfs:///temp/my_dir", "hdfs::///new_dir");
```

Comment:
The formatter might have split this line. Let's add some form of indentation in this example code.